cvat/apps/engine/migrations/0020_remove_task_flipped.py

```diff
@@ -0,0 +1,125 @@
+# Generated by Django 2.1.7 on 2019-06-18 11:08
+
+from django.db import migrations
+from django.conf import settings
+
+from cvat.apps.engine.task import get_image_meta_cache
+from cvat.apps.engine.models import Job, ShapeType
+
+from PIL import Image
+
+import os
+
+
+def _flip_shape(shape, size):
+    if shape.type == ShapeType.RECTANGLE:
+        shape.points = [
+            shape.points[2], # xbr -> xtl
+            shape.points[3], # ybr -> ytl
+            shape.points[0], # xtl -> xbr
+            shape.points[1]  # ytl -> ybr
+        ]
+
+    for x in range(0, len(shape.points), 2):
+        y = x + 1
+        shape.points[x] = size['width'] - shape.points[x]
+        shape.points[y] = size['height'] - shape.points[y]
+
+
+def frame_path(db_task, frame):
+    task_dirname = os.path.join(settings.DATA_ROOT, str(db_task.id))
+    d1 = str(int(frame) // 10000)
+    d2 = str(int(frame) // 100)
+    path = os.path.join(task_dirname, 'data', d1, d2, str(frame) + '.jpg')
+    return path
+
+
+def _get_image_meta_cache_path(self):
```

Comment:
@bsekachev , I believe we have the information in DB at the moment. Do we need to use the function? I hope to delete image_cache file in the future.

Reply:
@nmanovic 
Just I saw at the code of dump and it uses this cache. As far as I understand in db we have information for images, but not for videos.